README.md

```
@@ -168,6 +168,123 @@ See `.env.example` for complete list.
+
+```bash
+# NFT Contract Configuration
+NFT_CONTRACT_ADDRESS=0x...          # Set after deployment
+NFT_CHAIN_ID=1                      # 1 = Mainnet, 11155111 = Sepolia, 31337 = Local
+NFT_SIGNER_PRIVATE_KEY=0x...        # Backend signer private key (NEVER COMMIT)
+NFT_SIGNER_ADDRESS=0x...            # Public address of signer
+NFT_BASE_URI=https://babylon.market/api/nft/metadata/
+```
+
+Generate a signer keypair for production:
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)